api.ts

```diff
@@ -293,6 +294,12 @@ export interface SyncTableDef<
    * @hidden
    */
   namedPropertyOptions?: SyncTablePropertyOptions;
+
+  /**
+   * See {@link SyncTableOptions.roles}
+   * @hidden
+   */
+  roles?: TableRole[];
```

Comment:
Per the other discussion it seems like it was unnecessarily complex to make this an array without a clear use case.

Reply:
Moved back to being singular!